driver-ui-desktop/build.gradle

```diff
@@ -1,5 +1,8 @@
 dependencies {
     api driver_ui
+
+    // <ByImage, SikuliWebDriver>
+    implementation 'org.sikuli:sikuli-api:1.2.0'
```

Comment:
Sikuli is already part of `driver-ui/build.gradle`, Why it's needed here?

Reply:
You are right, this implementation is redundant and not needed. It has been removed in the most recent commit.